frontend/src/lib/near-social.ts

```
@@ -0,0 +1,40 @@
+        params: {
+          methodName: "set",
+          args: { data },
+          gas: "300000000000000",
+          deposit: "0",
+        },
+      },
+    ];
+
+    return { actions };
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)